presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -60,6 +60,7 @@
     EXCEEDED_FUNCTION_MEMORY_LIMIT(0x0000_0025, USER_ERROR),
     ADMINISTRATIVELY_KILLED(0x0000_0026, USER_ERROR),
     INVALID_COLUMN_PROPERTY(0x0000_0027, USER_ERROR),
+    QUERY_TOO_MANY_STAGES(0x0000_0028, USER_ERROR),
```

Comment:
How about we name this as `MAX_STAGE_COUNT_EXCEEDED`? This will make this error code sound more consistent with the other error codes, and also will make it consistent with the config/session property name.

Reply:
I see
 
```
QUERY_TEXT_TOO_LARGE
PAGE_TOO_LARGE
TOO_MANY_REQUESTS_FAILED
```

and
```
EXCEEDED_FUNCTION_MEMORY_LIMIT
EXCEEDED_MEMORY_LIMIT
EXCEEDED_TIME_LIMIT
EXCEEDED_CPU_LIMIT
EXCEEDED_SPILL_LIMIT
```

I can do either `QUERY_TOO_MANY_STAGES` or `EXCEEDED_STAGE_COUNT_LIMIT`. I did the former because I find this one more similar in nature to `PAGE_TOO_LARGE`. But I don't feel strongly.